src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -298,21 +298,21 @@ const UsersFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     justifyContent: 'center'
   },
   noDataFoundText: {
     alignSelf: 'center',
     justifyContent: 'center',
     fontFamily: fonts.inter[600],
-    color: COLORS.white2
+    color: COLORS.white
   },
   unfollowedHeaderContainer: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of `unfollowedHeaderContainer` has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Confirm that this change is in line with your design specifications.
```diff
-    backgroundColor: COLORS.gray100
+    backgroundColor: COLORS.lightgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->